monitor/nuro-data-collector/src/main/java/eu/seaclouds/monitor/nuroDc/MainDc.java

```diff
@@ -0,0 +1,83 @@
+package eu.seaclouds.monitor.nuroDc;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MainDc {
+    private static final Logger logger = LoggerFactory.getLogger(MainDc.class);
+
+    public String managerIp = null;
+
+    public String managerPort = null;
+
+    public String configFilePath = null;
+
+    private static final String DEFAULT_MANAGER_IP = "localhost";
+    private static final String DEFAULT_MANAGER_PORT = "8170";
+    private static final String DEFAULT_CONFIG_FILE_PATH = "config.properties";
+    private static final String ENV_VAR_MANAGER_IP = "MANAGER_IP";
+    private static final String ENV_VAR_MANAGER_PORT = "MANAGER_PORT";
+    private static final String ENV_VAR_URL_CONFIG_FILE = "NURODC_CONFIG_FILE";
+
+    public static void main(String[] args) throws Exception {
+
+        MainDc mainInstance = new MainDc();
+
+        Properties nuroDCProp = new Properties();
+
+        // load default values
+        mainInstance.loadDefaultValues();
+
+        // try to load from environment variables
+        mainInstance.loadFromEnrivonmentVariables();
+
+        // load properties from the config file
+        try {
+            nuroDCProp.load(new FileInputStream(mainInstance.configFilePath));
```

Comment:
what is the semantic of these different config strategies?

You are using `environment variables` and a config file. Would it be possible to use just one mechanism? I think this would simplify the bootstrap of the DC and its runtime management. Wdyt?